.github/workflows/tests.yml

```diff
@@ -46,13 +46,54 @@ jobs:
       - name: Run unit tests
         run: go test -v ./...
 
+  old_rpm_build:
+    name: "📦 RPM (golang-github-osbuild-composer)"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        fedora_release: ["31"]
+    container:
+      image: "docker.io/library/fedora:${{ matrix.fedora_release }}"
+    steps:
+      - name: "📥 Prepare container"
+        run: |
+          echo "fastestmirror=1" >> /etc/dnf/dnf.conf
+          echo "install_weak_deps=0" >> /etc/dnf/dnf.conf
+          rm -fv /etc/yum.repos.d/fedora*modular*
+          dnf -y upgrade
+          dnf -y install dnf-plugins-core findutils git make rpm-build rpmdevtools
+
+      - name: "🗄️ Clone the repository"
+        uses: actions/checkout@v2
+
+      - name: "🛒 Install RPM build dependencies"
+        run: dnf -y builddep golang-github-osbuild-composer.spec
+
+      - name: "🛠️ Build RPMs"
+        run: |
+          mkdir rpms
+          make old-srpm
+          cp -av rpmbuild/SRPMS/*.rpm rpms/
+          make old-rpm
+          cp -av rpmbuild/RPMS/*/*.rpm rpms/
+
+      - name: "📤 Upload artifacts"
+        uses: actions/upload-artifact@v1
+        with:
+          name: rpms
+          path: rpms/
+
+      - name: "🔎 Test RPM installation"
+        run: dnf -y install $(ls rpms/*.x86_64.rpm)
+
   rpm_build:
-    name: "📦 RPM"
```

Comment:
There's still a mention of `golang-github-osbuild-composer` in the "Install RPM dependencies" section.

(Can't comment on the lines that are not in the diff.)

Reply:
Ah, I missed that, thanks!